mass_mailing_partner/__openerp__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     "name": "Link partners with mass-mailing",
-    "version": "8.0.2.0.0",
+    "version": "8.0.3.0.0",
```

Comment:
No, it's not a big change and even doesn't require migration script, so put 2.1.0